odkx-src/basics-install.rst

```diff
@@ -30,12 +30,13 @@ We also recommend installing both ODK-X Survey and ODK-X Tables. Having both is
 Installing Services
 --------------------------------
 
-  1. From your device's :guilabel:`Settings`, choose :menuselection:`Security`.
+  1. From your device's :guilabel:`Settings`, choose :menuselection:`Apps & notifications`. (On older versions of Android, this setting maybe in :menuselection:`Applications` or :menuselection:`Security` depending upon your android version.)
 
-    - Make sure *Unknown Sources* is checked.
-    - (On older versions of Android, this setting is in :menuselection:`Applications` rather than :menuselection:`Security`)
+    - Go to :guilabel:`Special app access` in :guilabel:`Advanced` and choose :guilabel:`Install unknown apps`
+    - From the list of applications, select a browser of your choice and check :guilabel:`Allow from this source`.
+    - (On older versions of Android, above two steps are not required, make sure installation from *Unknown Sources* is checked.)
 
-  2. Open a web browser on your phone.
+  2. Open the same (*any* in case of older android versions) web browser on your phone.
```

Comment:
I am being a bit picky here, but i think it would be less confusing it was written as 
Open the same web browser that you authorized to install unknown apps on your phone. (For older versions of Android any web browser can be used since you do not need to specifically authorize the web browser's ability to install.)

Reply:
Sure no problem, I see how it will be a bit confusing for someone new reading it. I'll update it.